Remove XMeter Cloud from documents (#2817)
emqtt/emq-docs-cn: en_US/performance/performance-reference.md

# Test Scenarios and Results for Reference

This page provides an in-depth analysis of the performance of the EMQX in various scenarios by using XMeter to conduct performance tests. We explore EMQX's capabilities under different conditions, including varying Quality of Service (QoS) levels, Payload sizes, publish-subscribe models, and the impact of MQTT message bridging. Through rigorous testing and measurement, we aim to offer insights into EMQX's behavior and help users optimize its deployment for IoT applications.

## Test Environment

> All tests in this section are based on the open-source edition of **EMQX v5.1.6** deployed on a single node. EMQX and XMeter have created a peering connection to eliminate the interference of external network latency. The specifications of the server running EMQX are as follows.

Author: ```suggestion
All tests in this section are based on the open-source edition of **EMQX v5.1.6** deployed on a single node. A peering connection has been created between the EMQX and the testing server to eliminate the interference of external network latency. The specifications of the server running EMQX are as follows.
```

- **Maximum Packets Per Second**: 800,000 PPS
- **OS**: CentOS 7.9

> Except for the fan-in scenario where XMeter used 20 test clients to send and receive messages, the number of test clients in other scenarios was 10.

Author: ```suggestion
Except for the fan-in scenario where 20 test clients are used to send and receive messages, the number of test clients in other scenarios was 10.
```
